### PR TITLE
[charts/portal] - ingress-nginx readme correction

### PR DESCRIPTION
**Pull request by premchandervodela**
**Description of the change**

**Benefits**

**Drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[charts/gateway]`)
- [] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

**Comment by premchandervodela** [comment]
Incorporated review comment provided by Kiran V in PR 112.

File: charts/portal/README.md [hunk 1]
```diff
@@ -502,23 +502,22 @@ The following table lists the configured parameters of the MySQL Subchart - http
...
-| `nginx-ingress.ingressClassResource.default` | Creating the IngressClass- with the name specified, by default | `false` |
-| `nginx-ingress.ingressClassResource.controllerValue` | Creating the IngressClass- with the contollerValue specified | `k8s.io/ingress-nginx` |
+| `ingress-nginx.podSecurityPolicy.enabled`                | Tell Nginx to read PSP   | `true` |
+| `ingress-nginx.rbac.create`                | Create & use RBAC resources   | `true` |
+| `ingress-nginx.controller.publishService.enabled`                | Enable Publish Service   | `true` |
+| `ingress-nginx.controller.extraArgs.enable-ssl-passthrough`                | Enable SSL Passthrough   | `true` |
+| `ingress-nginx.controller.ingressClassResource.name` | Creation of the IngressClass- with the name | `nginx` |
+| `ingress-nginx.controller.ingressClassResource.enabled` | Creating the IngressClass- with the name specified, if this flag is enabled | `true` |
+| `ingress-nginx.controller.ingressClassResource.default` | Creating the IngressClass- with the name specified, by default | `false` |
```
Comment (ksaladi) [inline charts/portal/README.md]: For property "ingressClassResource.default" how about writing as  ' true value will makes this the default ingressClass for the cluster' and default value is false.  
For property "ingressClassResource.controllerValue" how about writing as 'value of the controller that is processing this ingressClass'
Picked from:- https://github.com/kubernetes/ingress-nginx/blob/c1be3499eb98756af4d2f5a5d165e6ff11cceeb5/charts/ingress-nginx/values.yaml#L104

or for more details, we can refer https://kubernetes.github.io/ingress-nginx/#getting-started

**Review by ksaladi — CHANGES_REQUESTED** [review]
Please see my comments

File: charts/portal/README.md (hunk 1 above)
Comment (premchandervodela) [reply charts/portal/README.md]: addressed the review comment

**Review by ksaladi — APPROVED** [review]
LGTM

**Pull request merged**